article/_posts/2021-09-02-sindyautoencoder.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: review
+title: "Data-driven discovery of coordinates and governing equations"
+tags: dimensionality-reduction autoencoder
+author: "Thierry Judge"
+cite:
+    authors: "Kathleen Champion, Bethany Lusch, J. Nathan Kutz1, Steven L. Brunton"
+    title: "Data-driven discovery of coordinates and governing equations"
+    venue: "Proceedings of the National Academy of Sciences of the United States of America"
+pdf: "https://arxiv.org/pdf/1904.02107.pdf"
+---
+
+# Introduction
+
+Finding governing equations for dynamical systems is an essential task in many scientific fields. However, these equations 
+are often unknown. With the increased availability of large datasets for time series data, many data-driven model 
+discovery approaches have been proposed. However, to find interesting models many of these methods required the data to 
+be in specific coordinate systems which is often not the case. 
+
+Authors propose a method to discover both the intrinsic coordinates systems and governing equations of high dimensional 
+data using a combination of Auto-encoders and Sparse identification of nonlinear dynamics (SINDy)[^1]. 
+  
+
+# Methods
+
+## Sparse identification of nonlinear dynamics (SINDy)
+
+The goal of the SINDy algorithm is to find a parsimonious dynamical model from time-series data. 
+
+Given snapshot data $$x(t) \in \mathcal{R}^n$$, the goal is to find a function $$\mathbb{f}$$ that fits a dynamical system
+
+$$
+\frac{d}{dt} x(t) = \mathbb{f}(x(t))
+$$
+
+Given $$m$$ data samples of length $$n$$, data snapshots and derivative snapshots are stacked to create 
+$$X, \dot{X} \in \mathcal{R}^{m \times n}$$. This method assumes the snapshot derivatives $$\dot{x}(t)$$ are available 
+or can be calculated from the data. 
+
+![](/article/images/sindyautoencoder/XXdot.jpg)
+
+Using a library of $$p$$ candidate basis functions $$\Theta(X) = [\theta_1(X), \theta_2(X), ..., \theta_p(X) ]$$ 
+(functions such as polynomials and trig. functions), $$\mathbb{f}$$ can be approximated.
+
+$$
+\dot{X} = \Theta(X)\Xi
+$$
+
+where $$\Xi = (\xi_1, \xi_2, ...\xi_n) \in \mathcal{R}^{p \times n}$$ is the set coefficients for each candidate function. 
+
+The goal of this method is to find a parsimonious model by using a sparse regression to find $$\Xi$$: 
+
+$$
+min_\Xi ||\dot{X} - \Theta(X)\Xi||_2^2 + \lambda ||\Xi||_1
+$$
+
+## SINDy Autoencoders
+
+SINDy works best with low dimensional data. To work with high dimensional data, such as videos, authors present, SINDy 
```

Comment:
present SINDy (w/o the comma, which creates a weird break here and makes the sentence harder to understand).